app/cdap/components/AddTransformation/CircularProgressBar/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { createStyles, Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles((_theme: Theme) => ({
+  progress: {
+    position: 'relative',
+    margin: '4px',
```

Comment:
we can just write margin: 4, instead of string margin: '4px'
plz check across the changes and fix it

Reply:
Done